server/controllers/chartsController/helpers.js

```diff
@@ -251,41 +256,83 @@ export const postProcessData = (data, studyTotals) => {
   return processedDataBySite
 }
 
-export const mongoQueryFromFilters = (filters) => {
+export const mongoQueriesFromFilters = (filters, userAccess) => {
   if (!filters) {
     return
   }
   const activeFilters = []
-  const facet = {}
+  const facetForFilters = {}
+  const includedCriteriaFacet = {}
   const chrCritFilters = filters.chrcrit_part
     .filter((f) => f.value === TRUE_STRING)
     .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
   const includedExcludedFilters = filters.included_excluded
     .filter((f) => f.value === TRUE_STRING)
     .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
+  const sexAtBirthFilters = filters.sex_at_birth
+    .filter((f) => f.value === TRUE_STRING)
+    .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
 
   if (!!chrCritFilters.length) {
-    facet.chrcrit_part = []
-    facet.chrcrit_part.push({
+    includedCriteriaFacet.chrcrit_part = []
+    includedCriteriaFacet.chrcrit_part.push({
       $match: { chrcrit_part: { $in: chrCritFilters } },
     })
     activeFilters.push('chrcrit_part')
   }
 
   if (!!includedExcludedFilters.length) {
-    facet.included_excluded = []
-    facet.included_excluded.push({
+    includedCriteriaFacet.included_excluded = []
+    includedCriteriaFacet.included_excluded.push({
       $match: { included_excluded: { $in: includedExcludedFilters } },
     })
     activeFilters.push('included_excluded')
   }
 
+  if (!!chrCritFilters.length || !!includedExcludedFilters.length) {
```

Comment:
what is an included excluded filter?

Reply:
This belongs to the Inclusion criteria assessment and the name of the variable is included_excluded. I don't know what it means behind the scenes, but I can find out.